src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -185,6 +185,9 @@ function newRepeatableElement(container, field_group, values) {
                     $(el).trigger('backpack_field.deleted');
                 });
                 $(this).parent().remove();
+
+                //we reassure row numbers
+                setupElementRowsNumbers(container_holder);
```

Comment:
Add `setupDependencySelector` if there are dependencies. So that the field uses this CUSTOM selector with row number and repeatable-name, not the default one. Store it as a data attribute (`data-dependency-selector`).